### PR TITLE
fix(node/fs): add sync method to filehandle method

### PR DESCRIPTION
**Pull request by yuxi-ovo**
links #25554

### CODE REVIEW

**Comment by yuxi-ovo** [comment]
fixes #25554

File: ext/node/polyfills/fs.ts [hunk 1]
```diff
@@ -189,6 +189,7 @@ const promises = {
   appendFile: appendFilePromise,
   readFile: readFilePromise,
   watch: watchPromise,
+  sync: fsyncSync,
```
Comment (kt3k) [inline ext/node/polyfills/fs.ts:192]: `promises.sync` doesn't seem existing in Node.js fs docs https://nodejs.org/api/fs.html

Or am I missing something?

Comment (yuxi-ovo) [reply ext/node/polyfills/fs.ts:192]: image
Is this correct? I noticed that the filehandle class in your issue requires this method. Did I misunderstand? Thank you for your reply

Comment (yuxi-ovo) [reply ext/node/polyfills/fs.ts:192]: Oh, this is my problem. Do I need to delete it now？
Thank you for your reply

Comment (marvinhagemeister) [reply ext/node/polyfills/fs.ts:192]: yes

Comment (yuxi-ovo) [reply ext/node/polyfills/fs.ts:192]: ok, I'm done. Thank you for your reply

Comment (marvinhagemeister) [reply ext/node/polyfills/fs.ts:192]: Thanks for taking a look! Maybe a commit wasn't pushed correctly. This line here is still included in the changes of the PR.

Comment (yuxi-ovo) [reply ext/node/polyfills/fs.ts:192]: here [`be61bce` (#27089)](https://github.com/denoland/deno/pull/27089/commits/be61bcebf0c1a062a60b6c83b78a9e8c9db04752)
Is this correct ？
thank you for your replay

**Pull request closed without merging**